src/components/Drawer/Drawer.tsx

```diff
@@ -103,26 +117,9 @@ export const Drawer = (props) => {
     );
 };
 
-Drawer.propTypes = {
-    /** Drawer title */
-    title: PropTypes.string.isRequired,
-    /** Set open/close status for initial state */
-    initialIsExpanded: PropTypes.bool,
-    /** Control visibility Drawer component */
-    isShown: PropTypes.bool,
-    /** Set open/close status for state */
-    isExpanded: PropTypes.bool,
-    /** Fired after click open/close button */
-    onClick: PropTypes.func,
-    /** Node(s) to be rendered inside the container */
-    children: PropTypes.node.isRequired,
-};
-
 Drawer.defaultProps = {
     initialIsExpanded: false,
```

Comment:
Should these two `defaultProps` still stay if you removed all of the `propTypes`?

Reply:
yes, we still need to use `propTypes` ](https://github.com/textkernel/oneui/blob/master/CONTRIBUTING.md#type-definitions-for-props-and-states), otherwise StoryBook doesn't pick up it.